swift-sdk/uicomponents/IterableEmbeddedView.swift

```diff
@@ -136,6 +136,12 @@ public class IterableEmbeddedView:UIView {
         return view
     }
     
+    public init(frame: CGRect, message: IterableEmbeddedMessage, viewType: IterableEmbeddedViewType, config: IterableEmbeddedViewConfig) {
```

Comment:
@evantk91 I think we can leave the frame off of this, and just pass CGRect (or nothing?) to super?